RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h

```diff
@@ -48,8 +48,8 @@ class PFCheckHitPattern {
   /// on track.
 
   PFTrackHitFullInfo 
-    analyze(edm::ESHandle<TrackerGeometry>, const reco::TrackBaseRef track, 
-	    const TransientVertex& vert);
+    analyze(const TrackerTopology* tkerTopo, const TrackerGeometry* tkerGeom,
+            const reco::TrackBaseRef track, const TransientVertex& vert);
```

Comment:
I'm not sure there's a clear recommendation, but I understood they are essentially equivalent (`edm::ESHandle` throws an error on access when the product can't be found, if I read http://cmsdoxygen.web.cern.ch/cmsdoxygen/CMSSW_9_2_0_patch3/doc/html/d0/d69/FWCore_2Framework_2interface_2ESHandle_8h_source.html correctly, so passing the raw pointer only makes this exception show up earlier) - I personally prefer the raw pointer because it makes the interface lighter (and clearer), and I wanted to be consistent, so changed the `TrackerGeometry` handle as well - but I may have missed something.